posthog/models/cohort/cohort.py

```diff
@@ -16,6 +16,20 @@
 from posthog.models.property import BehavioralPropertyType, Property, PropertyGroup
 from posthog.models.utils import sane_repr
 from posthog.settings.base_variables import TEST
+from prometheus_client import Counter
+from posthog.metrics import LABEL_TEAM_ID
+
+
+COHORT_ERRORED_CALCULATIONS_COUNTER = Counter(
+    "cohort_errored_calculations_total",
+    "Count of cohort calculations that failed",
+    labelnames=[LABEL_TEAM_ID, "cohort_id"],
+)
+COHORT_SUCCESSFUL_CALCULATIONS_COUNTER = Counter(
+    "cohort_successful_calculations_total",
+    "Count of cohort calculations that succeeded",
+    labelnames=[LABEL_TEAM_ID, "cohort_id"],
```

Comment:
I'm not sure cohort_id as a label is a good idea. Per documentation you shouldn't use too granular labels, as there will be an internal counter for all combinations. We have `team_id` for a few counters, which I think is as granular as is acceptable.

https://prometheus.io/docs/practices/naming/#labels

> CAUTION: Remember that every unique combination of key-value label pairs represents a new time series, which can dramatically increase the amount of data stored. Do not use labels to store dimensions with high cardinality (many different label values), such as user IDs, email addresses, or other unbounded sets of values.

Reply:
Thanks for flagging! will remove it